src/components/BeaconsForm.tsx

```diff
@@ -1,7 +1,7 @@
 import { useRouter } from "next/router";
 import React, { FunctionComponent, ReactNode } from "react";
 import { FormError } from "../lib/form/formManager";
-import { BackButton, BackButtonRouterIndexes, Button } from "./Button";
```

Comment:
`BackButtonRouterIndexes` was handling business logic such as "If user hasn't specified a BeaconUse, redirect to page X" a few levels deep in the React call tree.  Replaced with a `Rule`.